modules/nw-ptp-configuring-linuxptp-services-dual-port-oc.adoc

```diff
@@ -0,0 +1,100 @@
+// Module included in the following assemblies:
+//
+// * networking/ptp/configuring-ptp.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="configuring-linuxptp-services-oc-dual-port_{context}"]
+= Configuring linuxptp services as an ordinary clock with dual-port NIC redundancy
+
+You can configure `linuxptp` services (`ptp4l`, `phc2sys`) as an ordinary clock with dual-port NIC redundancy by creating a `PtpConfig` custom resource (CR) object.
```

Comment:
Should sentence two join with sentence one to form a paragraph?

Reply:
This is ventilated prose for greater readability and easier diffs. There's one sentence on one line, but they do form a paragraph in the output.